README.md

```diff
@@ -1,2 +1,129 @@
 # sexual-health-information-etl
-ETL to retrieve sexual health information and support services from syndication and store as JSON
+> ETL to retrieve sexual health information and support services from syndication and store as JSON
+
+[![GitHub Release](https://img.shields.io/github/release/nhsuk/sexual-health-information-etl.svg)](https://github.com/nhsuk/sexual-health-information-etl/releases/latest/)
+[![Greenkeeper badge](https://badges.greenkeeper.io/nhsuk/sexual-health-information-etl.svg)](https://greenkeeper.io/)
+[![Build Status](https://travis-ci.org/nhsuk/sexual-health-information-etl.svg?branch=master)](https://travis-ci.org/nhsuk/sexual-health-information-etl)
+[![Coverage Status](https://coveralls.io/repos/github/nhsuk/sexual-health-information-etl/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/sexual-health-information-etl?branch=master)
+[![Known Vulnerabilities](https://snyk.io/test/github/nhsuk/sexual-health-information-etl/badge.svg)](https://snyk.io/test/github/nhsuk/sexual-health-information-etl)
+
+## Installation
+
+Clone the repo: `git clone https://github.com/nhsuk/sexual-health-information-etl.git`
+and review the [`scripts`](scripts) to get up and running.
+
+## Testing
+
+The application uses [Docker](https://www.docker.com/) to run in containers.
+Development is typically done on the host machine. Files are loaded into the
+container and changes are automatically updated.
+
+Use the `test` script for continuous testing during development.
+
+# Run process
+
+In order for the process to access the syndication feed an API key is required.
+Details of registration are available on
+[NHS Choices](http://www.nhs.uk/aboutNHSChoices/professionals/syndication/Pages/Webservices.aspx).
+The application needs the API key available within the environment as the variable `SYNDICATION_API_KEY`.
+
+The output is uploaded to Azure Blob Storage, a suitable connection string should be set in the `AZURE_STORAGE_CONNECTION_STRING` variable.
+For further details see [Azure Blob Storage](https://azure.microsoft.com/en-gb/services/storage/blobs/).
+
+The ETL retrieves all the Sexual Health and Information Services from the Syndication API via the `modifiedsince` end point.
+Serco reload all the data when a change is made, so the `modifiedsince` end point will contain all records.
```

Comment:
In this case modifiedsince is used to capture when a data change occurs - it will always contain all the data, and an incremental update would not be possible due to the IDs changing each time. I'll improve the comment to highlight the fact that data is reloaded with different IDs from the previous load, which prevents a usual incremental update.